spec/rubocop/cop/lint/number_conversion_spec.rb

```diff
@@ -108,4 +108,55 @@
       RUBY
     end
   end
+
+  context 'autoccorrection' do
```

Comment:
you can add `expect_correction()` to the existing offending examples

Reply:
It's in indeed more elegant.